message FaultCoverageExperimentData{

	optional string data_name = 1;
	required int64 m_InstrPtr1 = 2;
	required int64 m_InstrPtr2 = 3;

}
